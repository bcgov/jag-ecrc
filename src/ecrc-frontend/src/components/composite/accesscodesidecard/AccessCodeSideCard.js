import React from "react";

import SideCard from "../../base/sideCard/SideCard";

export default function AccessCodeSideCard() {
  const accessCode = {
    heading: "Need help with an access code?",
    content: [
      "Access code is specific to each organization. If you need help with the organization access code, please ",
      <b>contact your organization.</b>
    ],
    type: "bluegrey"
  };

  return <SideCard sideCard={accessCode} />;
}
