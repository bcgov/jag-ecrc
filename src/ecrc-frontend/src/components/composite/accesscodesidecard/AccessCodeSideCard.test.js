import React from "react";
import { create } from "react-test-renderer";

import AccessCodeSideCard from "./AccessCodeSideCard";

describe("AccessCodeSideCard Component", () => {
  test("Matches the snapshot", () => {
    const accessCode = create(<AccessCodeSideCard />);
    expect(accessCode.toJSON()).toMatchSnapshot();
  });
});
