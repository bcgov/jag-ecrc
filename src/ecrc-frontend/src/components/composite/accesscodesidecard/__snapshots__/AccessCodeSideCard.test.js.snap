// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccessCodeSideCard Component Matches the snapshot 1`] = `
<div
  className="dashboard-spacing"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col-12"
      style={
        Object {
          "paddingTop": "30px",
          "position": "relative",
        }
      }
    >
      <p />
      <section
        className="bluegrey-container"
        id="bluegrey-section"
      >
        <div
          className="container-background bluegrey-heading"
        >
          <h2
            className="heading-style"
          >
            Need help with an access code?
          </h2>
        </div>
        <div
          className="bluegrey-content"
        >
          <p
            className="content-style"
          >
            Access code is specific to each organization. If you need help with the organization access code, please 
            <b>
              contact your organization.
            </b>
          </p>
        </div>
      </section>
    </div>
  </div>
</div>
`;
